WHID_Elite_HybridCC_v.1.0/README.md

```diff
@@ -0,0 +1,33 @@
+# Hybrid C&C Workflow SMS & GPRS
```

Comment:
Why the mousejacking and the RF commands were removed?
Not enough memory left after compiling the sketch?